src/cfn.py

```diff
@@ -202,16 +202,10 @@ def cmd(stackname, command=None, username=DEPLOY_USER):
     print "Connecting to: %s" % stackname
     stack_all_ec2_nodes(
         stackname,
-        cmd_parallel_subtask,
+        (parallel(lambda command: run(command)), {'command': command}),
```

Comment:
actually, it caused a linting issue and I improved it even further after taking a look at the innards of `execute`. 

I wouldn't call this idiomatic though ... `lambda` and functional programming are second class citizens in Python these days